.travis.yml

```
@@ -20,4 +20,4 @@ install:
 script:
   - python -m isort -c --skip-glob=venv
   - python -m flake8 tests/ validate_email/ setup.py
-  - python -m unittest discover -v
+  - python -Im unittest discover -v -s tests
```

```suggestion
  - python -m unittest discover -v -s tests
```
`-I` isolates the user environment (`~/.local/lib...` under linux), not the virtual env you intended, I think.